engines/drascula/drascula.cpp

```diff
@@ -249,6 +250,18 @@ Common::Error DrasculaEngine::run() {
 		_lang = kEnglish;
 	}
 
+	Common::TextToSpeechManager *ttsMan = g_system->getTextToSpeechManager();
+	if (ttsMan != nullptr) {
+		ttsMan->setLanguage(ConfMan.get("language"));
+		ttsMan->enable(ConfMan.getBool("tts_enabled"));
+
+		if (_lang == kRussian) {
+			_ttsTextEncoding = Common::CodePage::kWindows1251;
+		} else {
+			_ttsTextEncoding = Common::CodePage::kUtf8;
```

Comment:
That's sound fishy. The text in the game datafiles for western European languages uses CP 850 encoding. So I would expect `Common::CodePagekDos850` here and not `Common::CodePage::kUtf8`.